### DISCUSSION TITLE
PovertyMap Nighttime Lights Images

### DISCUSSION THREAD

**Discussion by niki-amini-naieni** [p-lambda/wilds#115] [Q&A] [answered]
Hello,

Firstly, thank you for providing the WILDS benchmark. It has been really helpful for testing out novel algorithms on real-world datasets.

I am wondering if you could provide screenshots of the unnormalized versions of the nighttime lights channel from the PovertyMap dataset. When I attempt to visualize the nighttime lights, the images seem quite pixelated (as if there is some nearest neighbor interpolation occurring even though the images themselves are 224 x 224 pixels). However, I am able to reproduce the RGB images from the paper shown below by unnormalizing the RGB channels of the preprocessed images. The resulting images are not pixelated like the nighttime lights channels are. 
<img width="674" alt="image" src="https://user-images.githubusercontent.com/61301874/159139087-2f7542be-4bc7-42f1-909c-69710c1bc17e.png">

Thank you again for all your help!

**Comment by sangmichaelxie** [comment, marked as answer]
This seems correct - the nighttime light channels are from a different source and should be of a different resolution than the images.

**Reply by niki-amini-naieni** [reply]
Got it, thanks!
